docs/pages/user_manual.md

```diff
@@ -1,86 +1,154 @@
 # ModbusScope - User Manual
 
-This user manual describes how to get started with *ModbusScope*.
+This user manual describes how to get started with *ModbusScope*. *ModbusScope* is an application which can sample Modbus registers and plot the value in a graph to visualize and investigate the dynamic behavior of a system.
 
-## Installing
+## Overview
```

Comment:
A quick start section could help people get up-and-running quickly. Something along the lines of:

```md
## Quickstart

A quick start to use modbusscope:

* Add a register using the Register Settings
* Press play to start logging
* Press stop 

```

Reply:
Added to doc